package.json

```diff
@@ -85,7 +85,7 @@
     "@qwant/merge-po": "file:local_modules/merge-po",
     "@qwant/nconf-builder": "file:local_modules/nconf_builder",
     "@qwant/po-js": "file:local_modules/po-js",
-    "@qwant/qwant-basic-gl-style": "git+https://github.com/QwantResearch/qwant-basic-gl-style.git#95e1495",
+    "@qwant/qwant-basic-gl-style": "git+https://github.com/QwantResearch/qwant-basic-gl-style.git#058d8351",
```

Comment:
This is now this commit you want :) https://github.com/QwantResearch/qwant-basic-gl-style/commit/2a50857c6f68e04ef2136c4034939ba2bc222284

Reply:
I'll be careful about this in the future, I wasn't sure it mattered! (I still miss some obvious notions of good practice in general!)